data/net/minecraft/client/sounds/SoundEngine.mapping

```diff
@@ -22,52 +31,156 @@ CLASS net/minecraft/client/sounds/SoundEngine
+		COMMENT   <li>Calculates and handles various sound properties such as volume, pitch, attenuation, looping, position and relative, </li>
+		COMMENT </ol>
+		COMMENT <p>
+		COMMENT @implNote This method assumes proper synchronization or thread confinement mechanisms are in place.
 		ARG 1 sound
+			COMMENT The sound instance to be played.
 	METHOD playDelayed (Lnet/minecraft/client/resources/sounds/SoundInstance;I)V
 		COMMENT Adds a sound to play in n tick
 		ARG 1 sound
 		ARG 2 delay
 	METHOD queueTickingSound (Lnet/minecraft/client/resources/sounds/TickableSoundInstance;)V
+		COMMENT Queues a new {@linkplain TickingCodeInstance}
 		ARG 1 tickableSound
+			COMMENT the {@linkplain TickableSoundInstance} to add
+	METHOD reload ()V
+		COMMENT Reloads the sound engine.
+		COMMENT <p>
+		COMMENT This method clears the warning set, checks for missing sound events, destroys the current sound system,
+		COMMENT and reloads the library.
 	METHOD removeEventListener (Lnet/minecraft/client/sounds/SoundEventListener;)V
 		ARG 1 listener
 	METHOD requestPreload (Lnet/minecraft/client/resources/sounds/Sound;)V
+		COMMENT Requests to preload a specific {@linkplain Sound} instance
 		ARG 1 sound
 	METHOD requiresManualLooping (Lnet/minecraft/client/resources/sounds/SoundInstance;)Z
+		COMMENT Checks if a SoundInstance requires manual looping.
+		COMMENT <p>
+		COMMENT @return true if the SoundInstance requires manual looping, false otherwise
 		ARG 0 sound
+			COMMENT The SoundInstance to check
 	METHOD resume ()V
 		COMMENT Resumes playing all currently playing sounds (after pauseAllSounds)
+	METHOD shouldChangeDevice ()Z
+		COMMENT Checks if a change in the audio device should occur.
+		COMMENT <p>
+		COMMENT If the current audio device is disconnected, it logs an information message and returns true to indicate a change is needed.
+		COMMENT Otherwise, it checks if enough time has elapsed since the last device check.
+		COMMENT <p>
+		COMMENT If the elapsed time is greater than or equal to 1000 milliseconds, it performs the device check.
+		COMMENT During the device check, it compares the current device state with the preferred sound device specified in the options.
+		COMMENT <p>
+		COMMENT If the preferred sound device is an empty string and the system default audio device has changed, it logs an information message
+		COMMENT and sets the device pool state to indicate a change has been detected.
+		COMMENT <p>
+		COMMENT If the preferred sound device is not an empty string, it checks if the current device name is different from the preferred device name
+		COMMENT and if the preferred device is available in the list of available sound devices.
+		COMMENT <p>
+		COMMENT If both conditions are true, it logs an information message and sets the device pool state to indicate a change has been detected.
+		COMMENT <p>
+		COMMENT Finally, it sets the device pool state to indicate that the device check is complete.
```

Comment:
All of this should be rewritten in passive voice.

Reply:
Please rewrite it in Passive Voice :thinkies: